engine/runtime/src/main/scala/org/enso/interpreter/runtime/util/GitHubDiagnosticFormatter.scala

```diff
@@ -0,0 +1,154 @@
+package org.enso.interpreter.runtime.util
+
+import com.oracle.truffle.api.source.Source
+import org.enso.compiler.core.ir.Diagnostic
+
+import java.nio.file.Path
+import scala.annotation.tailrec
+
+/** An extension of [[DiagnosticFormatter]] that additionally prints commands for a GitHub workflow that will add annotations for each warning/error. */
+class GitHubDiagnosticFormatter(
```

Comment:
I know encapsulation isn't Scala's best attitude, but:
```suggestion
private[util] class GitHubDiagnosticFormatter(
```
would make me feel better. Isn't `DiagnosticsFormatter.make` (or rather `create`?) the right factory method to create an instance of formatter?

Reply:
That seems fair.